fuzzers/qemu_systemmode/src/fuzzer_classic.rs

```diff
@@ -84,9 +88,23 @@ pub fn fuzz() {
 
     let mut run_client = |state: Option<_>, mut mgr, _core_id| {
         // Initialize QEMU
-        let args: Vec<String> = env::args().collect();
-        let env: Vec<(String, String)> = env::vars().collect();
-        let qemu = Qemu::init(&args, &env).unwrap();
+        let qemu = Qemu::builder()
+            .machine(qemu_config::Machine::new("mps2-an385"))
+            .monitor(qemu_config::Monitor::null)
+            .kernel(qemu_config::Kernel::new(Path::new(
+                "target/classic/example.elf",
+            )))
+            .serial(qemu_config::Serial::null)
+            .no_graphic(qemu_config::NoGraphic::ENABLE)
```

Comment:
If all of them are either `ENABLE`(D) or `DISABLED`, why not make all of these take the same struct (or even just a bool?)

Reply:
i think bool is the right choice here. it's more consistent with other builders and keeps the same semantic.